CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Remove copyright year and sign from license and attributions
 
 ### Fixed
+- Fix homepage image links that were too large in Internet Explorer [styleguide-only]
```

Comment:
We should be specific on the version of Internet Explorer that was having an issue. Please add which versions of IE this is effected by and solves for.